app/server/api/user_api.py

```diff
@@ -125,7 +126,10 @@ def get(self, user_id):
     def post(self, user_id):
 
         post_data = request.get_json()
-        organisation = g.active_organisation
+        organisation = g.get('active_organisation')
+        if organisation is None:
+            # TODO: Refactor.. Query string needed for kobo
+            organisation = Organisation.query.get(request.args.get('org'))
```

Comment:
This is a short term fix

Reply:
Will a user ever not have an `active_organisation?

Reply:
no, but for external auth credentials organisation would need to be pulled from query string (insecure), until your PR is merged in https://github.com/teamsempo/SempoBlockchain/pull/140

Reply:
could just merge in your PR first ...